mediagrains/grain_constructors.py

```diff
@@ -43,26 +43,23 @@
 
 
 @overload
-def Grain(src_id_or_meta: GrainMetadataDict,
-          flow_id_or_data: GrainDataParameterType = None) -> GRAIN: ...
+def Grain(meta: GrainMetadataDict,
+          data: GrainDataParameterType = None) -> GRAIN: ...
 
 
 @overload
-def Grain(src_id_or_meta: Optional[UUID] = None,
-          flow_id_or_data: Optional[UUID] = None,
+def Grain(source_id: Optional[UUID] = None,
+          flow_id: Optional[UUID] = None,
           origin_timestamp: Optional[Timestamp] = None,
           sync_timestamp: Optional[Timestamp] = None,
           creation_timestamp: Optional[Timestamp] = None,
           rate: Fraction = Fraction(0, 1),
           duration: Fraction = Fraction(0, 1),
-          flow_id: Optional[UUID] = None,
           data: GrainDataParameterType = None,
-          src_id: Optional[UUID] = None,
-          source_id: Optional[UUID] = None,
           meta: Optional[GrainMetadataDict] = None) -> GRAIN: ...
 
 
-def Grain(src_id_or_meta=None,
+def Grain(source_id_or_meta=None,
           flow_id_or_data=None,
```

Comment:
Not directly related to this commit, but could these ..._or_... parameters be removed similarly to `src_id_or_meta` above?